semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -261,6 +255,57 @@ public static boolean isSqlTypeBackendForOtelFormat(
     return Optional.empty();
   }
 
+  public static Optional<String> getCassandraURI(Event event) {
```

Comment:
Please add a comment above the method, around the expected host value and why the need for trim before `:`